scylla-cql/src/types/serialize/writers.rs

```diff
@@ -74,6 +76,8 @@ impl<'buf> RowWriter<'buf> {
 /// in nothing being written.
 pub struct CellWriter<'buf> {
     buf: &'buf mut Vec<u8>,
+    cell_len: Option<usize>,
+    size_as_uvarint: bool,
 }
```

Comment:
@Lorak-mmk As you know the serialization framework quite well, could you please aid in review of this commit?